app/interface/deps/scraper_deps.py

```
@@ -1,19 +1,17 @@
-async def get_kutis_scraper(
-    client: KutisClient = Depends(get_kutis_client),
-) -> AsyncGenerator[KutisScraper, None]:
-    yield KutisScraper(client=client)
-
-
-async def get_lms_scraper(
-    client: LmsClient = Depends(get_lms_client),
-) -> AsyncGenerator[LmsScraper, None]:
-    yield LmsScraper(client=client)
+get_barun_scraper = scraper_provider_factory(
+    get_barun_client, BarunParser, BarunScraper
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)